Automattic/jetpack
# Add rewind status to state tree

Add Rewind Stats to the state tree.

This PR includes the "status" of rewind for the given site.  It queries a new endpoint `/rewind` that will tell us is Rewind is active for the site.  It queries this endpoint at a very top level in `<Main />`, as we will need this in most of the children. 

Right now it affects very little UI. ONLY If the site's Rewind status is `active`, then it will display an Activity Log card.  This card will be refactored to represent Rewind in a later PR, as well as many other card.  (see #8266)

The endpoint response will look something like this: 
```
array(
	'reason' => 'host_not_supported',
	'state' => 'active', // Can also be 'available' or 'unavailable'
	'downloads' => array(),
	'last_updated' => 1513366050
);
```

To Test: 
- On any Jetpack admin page, you should see a `/rewind` endpoint being called in your Network tab. 
- Make sure the state is properly updated to reflect this endpoint by looking in `jetpack.rewind.data.status` in the state tree, or by checking `REWIND_STATUS_FETCH_RECEIVE` in your Redux Dev tools. 
![screen shot 2017-12-18 at 3 45 39 pm](https://user-images.githubusercontent.com/7129409/34127319-aa0746d0-e40a-11e7-809e-5b8e6c27810d.png)
